### PR TITLE
[kokoro] Rewrite swift MDF import statements for kokoro builds.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

File: .kokoro [hunk 1]
```diff
@@ -42,13 +42,16 @@ fix_bazel_imports() {
...
+  rewrite_source "s/import MDFTextAccessibility/import material_text_accessibility_ios_MDFTextAccessibility/"
+  rewrite_source "s/import MDFInternationalization/import material_internationalization_ios_MDFInternationalization/"
   stashed_dir="$(pwd)/"
   reset_imports() {
     # Undoes our source changes from above.
     rewrite_tests "s/import components_private_Math_Math/import MaterialComponents.MaterialMath/"
     rewrite_tests "s/import components_(.+)_.+/import MaterialComponents.Material\1/"
     rewrite_source "s/import \"Motion(.+)\.h\"/import <Motion\1\/Motion\1.h>/"
     rewrite_source "s/import \"MDF(.+)\.h\"/import <MDF\1\/MDF\1.h>/"
+    rewrite_source "s/import material_internationalization_ios_MDFInternationalization/import MDFInternationalization/"
```
Comment (romoore) [inline .kokoro]: I might be missing it, but I think the accessibility rewrite is needed in the `reset` function.

**Review by romoore — DISMISSED** [review]

File: .kokoro (hunk 1 above)
Comment (jverkoey) [reply .kokoro]: Done.

File: .kokoro [hunk 2]
```diff
@@ -42,13 +42,17 @@ fix_bazel_imports() {
...
+  rewrite_source "s/import MDFTextAccessibility/import material_text_accessibility_ios_MDFTextAccessibility/"
+  rewrite_source "s/import MDFInternationalization/import material_internationalization_ios_MDFInternationalization/"
   stashed_dir="$(pwd)/"
   reset_imports() {
     # Undoes our source changes from above.
     rewrite_tests "s/import components_private_Math_Math/import MaterialComponents.MaterialMath/"
     rewrite_tests "s/import components_(.+)_.+/import MaterialComponents.Material\1/"
     rewrite_source "s/import \"Motion(.+)\.h\"/import <Motion\1\/Motion\1.h>/"
     rewrite_source "s/import \"MDF(.+)\.h\"/import <MDF\1\/MDF\1.h>/"
+    rewrite_source "s/import material_text_accessibility_ios_MDFInternationalization/import MDFTextAccessibility/"
```
Comment (romoore) [inline .kokoro]: Copy-paste bug?

Comment (jverkoey) [reply .kokoro]: Aye. Fixed.

**Review by romoore — APPROVED** [review]

**Pull request merged**